src/module/actor/lancer-actor.ts

```diff
@@ -30,6 +31,9 @@ import type { RegEntry, MechWeapon, NpcFeature } from "machine-mind";
 import { StabOptions1, StabOptions2 } from "../enums";
 import { fix_modify_token_attribute } from "../token";
 import type { ActionData } from "../action";
+import { frameToPath } from "./retrograde-map";
+import { string } from "io-ts";
+import { NpcClass } from 'machine-mind';
```

Comment:
You don't appear to be actually using io-ts/`string`, and the import from mm can be an `import type`.